### PR TITLE
Updates and Chillx Extractor Updated

### PR DESCRIPTION
**Pull request by phisher98**
Updates and Chillx Extractor Updated

Kudos to @rushi-chavan for updated Keys

### CODE REVIEW

File: app/src/main/java/com/lagradost/cloudstream3/extractors/Chillx.kt [hunk 1]
```diff
@@ -24,34 +22,30 @@ class Watchx : Chillx() {
...
-                headers = mapOf(
-                    "Accept" to "text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,*/*;q=0.8",
-                    "Accept-Language" to "en-US,en;q=0.5",
-                )
+                referer = url,
             ).text
         )?.groupValues?.get(1)
-        val decrypt = cryptoAESHandler(master ?: return, getKey().toByteArray(), false)?.replace("\\", "") ?: throw ErrorLoadingException("failed to decrypt")
-
+        val key = app.get("https://raw.githubusercontent.com/rushi-chavan/multi-keys/keys/keys.json").parsedSafe<Keys>()?.key?.get(0) ?: throw ErrorLoadingException("Unable to get key")
```
Comment (fire-light42) [inline app/src/main/java/com/lagradost/cloudstream3/extractors/Chillx.kt:44]: Make this cached, we dont want to ddos github for every url.

**Review by fire-light42 — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/lagradost/cloudstream3/extractors/Chillx.kt (hunk 1 above)
Comment (phisher98) [reply app/src/main/java/com/lagradost/cloudstream3/extractors/Chillx.kt:44]: I have assigned that as key can be changed and previously as well its fetching from a Github json hosted on github so do you want me to assign key static in Extractor ?

Comment (fire-light42) [reply app/src/main/java/com/lagradost/cloudstream3/extractors/Chillx.kt:44]: @phisher98 What I mean is that you should do something like this, as then you dont have to do an extra get request for every time you use this extractor
```kotlin
companion object {
  private val key : String? = null
  suspend fun getKey() {
    key ?: app.get("...").parsedSafe<Keys>()?.key?.get(0)?.also { key = it }
  }
}

```

**Comment by b4byhuey** [comment]
private var key: String? = null

val key = fetchKey() ?: throw ErrorLoadingException("Unable to get key")
		
val decrypt = cryptoAESHandler(master ?: return, key.toByteArray(), false)
            ?.replace("\\", "")
            ?: throw ErrorLoadingException("failed to decrypt")

private suspend fun fetchKey(): String? {
        return app.get("https://raw.githubusercontent.com/rushi-chavan/multi-keys/keys/keys.json")
            .parsedSafe<Keys>()?.key?.get(0)?.also { key = it }
    }


@phisher98 try this

**Comment by phisher98** [comment]
@b4byhuey Yeah That what i was saying but Thanks for the help

**Comment by fire-light42** [comment]
The key is still not cached

**Review by fire-light42 — APPROVED** [review]

**Pull request merged**